docs/roadmap.rst

```diff
@@ -0,0 +1,88 @@
+Roadmap
+=======
+
+Process
+-------
+
+Read the Docs has adopted the following workflow with regards to how we
+prioritize our development efforts and where the core development team focuses
+its time.
+
+Triaging issues
+~~~~~~~~~~~~~~~
+
+Much of this is already covered in our guide on :doc:`contribute`, however to
+summarize the important pieces:
+
+* New issues coming in will be triaged, but won't yet be considered part of our
+  roadmap.
+* If the issue is a valid bug, it will be assigned the ``Accepted`` label and
+  will be prioritized, likely on an upcoming point release.
+* If the issues is a feature or improvement, the issue might go through a design
+  decision phase before being accepted and assigned to a milestone. This is a
+  good time to discuss how to address the problem technically. Skipping this
+  phase might result in your PR being blocked, sent back to design decision, or
+  perhaps even discarded. It's best to be active here before submitting a PR.
+* The core team will only work on accepted issues, and will only give PR review
+  priority to accepted issues.
+
+Milestones
+~~~~~~~~~~
+
+We maintain two types of milestones: point release milestones for our upcoming
+releases, and group milestones, for blocks of work that have priority in the
+future.
```

Comment:
Do you think that group milestones are good for future priority?

I think it's good to use them as a way to group related amount of work. The priority will be given by the `Accepted` label or when the issue is moved from a group milestone to a point release milestone.

Reply:
But we have certain planned features and groups of features that have a priority. For instance in the beginning of the year, this was build stability and admin ux -- and still is. This implies that work in these milestones will be first to be added to our releases, instead of randomly applying issues into our point releases.

Reply:
Yeah, that makes sense to me. I was wondering about when to add to any of this milestones. I suppose that we should add them if just "the topic matches" without considering any kind of priority.

Reply:
With some of our work shifting, and with point releases thinning out a little bit, we'll be leaking more issues into point releases now. It's been heavily leaning toward bug fixes for a while.